packages/editor/package.json

```diff
@@ -51,7 +51,7 @@
     "test": "jest"
   },
   "dependencies": {
-    "@apollo/client": "^3.3.21",
+    "@apollo/client": "^3.6.2",
```

Comment:
Fixes inconsitencies between `apollo-upload-client` that was ahead of the editor that caused a type error.